index.html

```diff
@@ -0,0 +1,121 @@
+                    </div>
+                    <div class="functionaliteit-item">
+                        <input type="text" class="form-input kern-functionaliteit" placeholder="Functionaliteit 3" required>
+                    </div>
+                </div>
+                <button type="button" id="addKernFunctionaliteit" class="btn-primary mt-2">+</button>
+            </div>
+
+
+
+            <!-- Backend & Fullstack Fields -->
+            <div id="backendFields" class="hidden">
+                <div>
+                    <label for="entities" class="form-label">Entiteiten</label>
+                    <p class="label-subtitle">Wat zijn entiteiten in Java? Een entiteit vertegenwoordigt een object in de database en lijkt op iets zoals je het in de echte wereld ook hebt, zoals gebruikers, producten, adressen, bestellingen, hotelkamers, etc. Deze vertalen we dan naar entiteiten met specifieke eigenschappen, zoals een hotelkamer bijvoorbeeld altijd een kamernummer, aantal slaapplaatsen, of het een roken/niet-roken kamer is en een prijs per nacht. Dit kun je dan als volgt opschrijven: Room(roomNumber, capacity, smokingAllowed, pricePerNightPerPerson)</p>
+                    <textarea id="entities" class="form-textarea" rows="5" placeholder="Elke entiteit op een nieuwe regel"></textarea>
+                </div>
+            </div>
+
+            <!-- Frontend Fields -->
+            <div id="frontendFields" class="hidden">
+                <div>
+                    <label class="form-label">Kies een backend</label>
+
+                    <p class="label-subtitle">Wij faciliteren de <a class="link" target="_blank"  href="https://novi.datavortex.nl">Novi Educational Backend.</a> dit is een database waar je gebruikers kan opslaan. Meer informatie staat op de website. Je mag ook een database gebruiken van eerdere leerlijn of een firebase gebruikers database. Deze zijn lastiger om te configureren en je krijgt er niet een extra punt voor.  </p>
+                    <select id="backendType" class="form-input">
+                        <option value="Novi's Educational backend">Novi's Educational backend</option>
+                        <option value="Eigen backend vanuit eerdere leerlijn">Eigen backend vanuit een eerdere leerlijn</option>
+                        <option value="Firebase">Firebase</option>
+                    </select>
+                </div>
+                <div>
+                    <label for="otherAPIs" class="form-label">Welke andere API ga je gebruiken?</label>
+                    <p class="label-subtitle"> Op <a class="link" target="_blank" href="https://github.com/hogeschoolnovi/frontend-api-list">Github</a> staat een lijst met API's die je mag gebruiken. Wil je een andere API gebruiken dan mag dat ook. Het is van belang dat je de API naam + URL hier inlever. zodat wij kunnen kijken of de api toegankelijk genoeg is om te gebruiken.</p>
```

Comment:
Kijk eens op [GitHub](https://github.com/hogeschoolnovi/frontend-api-list) om je te oriënteren op mogelijke API's die je kunt gebruiken voor een eigen idee. Deze lijst is door NOVI samengesteld en gecontroleerd. Wanneer je een andere API wil gebruiken, is het belangrijk om de link naar de documentatie te verschaffen zodat wij kunnen checken of de API bruikbaar is.